### PR TITLE
Update portals.md

### PR DESCRIPTION
**Pull request by msbemba**
Added AAD portal details to other workloads section

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7521

File: microsoft-365/security/defender/portals.md [hunk 1]
```diff
@@ -49,7 +49,8 @@ While these portals are not specifically for managing security, they support var
 
 | Portal name | Description | Link | 
 |---|---|---| 
-| Azure portal | View and manage [Azure Active Directory](/azure/active-directory/fundamentals/active-directory-whatis) and all your [Azure resources](/azure/azure-resource-manager/management/overview)  | [portal.azure.com](https://portal.azure.com/) |
+| Azure portal | View and manage all your [Azure resources](/azure/azure-resource-manager/management/overview)  | [portal.azure.com](https://portal.azure.com/) |
+| Azure active directory portal | View and manage [Azure Active Directory](/azure/active-directory/fundamentals/active-directory-whatis) | [aad.portal.azure.com](https://aad.portal.azure.com/) |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/portals.md]:
```suggestion
| Azure Active Directory portal | View and manage [Azure Active Directory](/azure/active-directory/fundamentals/active-directory-whatis) | [aad.portal.azure.com](https://aad.portal.azure.com/) |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**